src/sql/connection/connection.py

```diff
@@ -486,7 +486,7 @@ def dialect(self):
         pass
 
     @abc.abstractmethod
-    def raw_execute(self, query, parameters=None):
+    def raw_execute(self, query, parameters=None, enable_named_params="false"):
```

Comment:
Please `enable_named_params` if we dont need it

Reply:
Whoops, forgot about the abstract methods. [Removed](https://github.com/ploomber/jupysql/pull/980/commits/9b846cca357c300e2393536b718d03986dd84118#diff-62e3d6289e3242b271e6df3a7f201ab003f8ce73a5daf1a824cc2b4d892513d1R489)